src/components/ActionBar/ActionBarHeader.tsx

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import * as React from 'react';
+import cn from 'classnames';
+import { forwardRef } from 'react';
+import { Paragraph } from '@digdir/design-system-react';
+
+import { useActionBarContext } from './Context';
+import classes from './ActionBarHeader.module.css';
+import { ActionBarIcon } from './ActionBarIcon';
+import { ActionBarActions } from './ActionBarActions';
+
+export interface ActionBarHeaderProps {
```

Comment:
Pick all props that's defined in ActionBar
```suggestion
export interface ActionBarHeaderProps extends Pick<HeadingProps, 'level', 'title, 'subtitle', 'additionaltext', 'actions'>
{
```